tests/sql_mode/data/db2.prepare.sql

```diff
@@ -0,0 +1,10 @@
+drop database if exists `sql_mode`;
```

Comment:
We should also prepare some sql_mode specified data so we can test the batch load phase don't impact by different sql_mode

Reply:
In this pr, db1 is the global sql_mode enable and the session sql_mode disable; db2 is the opposite. And the selected test sql_modes are almost all samples that may cause errors. 

> https://pingcap.feishu.cn/docs/doccn61Fi9k5P8x22M5YnL1eNPh

PTAL, and what sql_mode specified data should I prepare?